osquery/tables/utility/time.cpp

```diff
@@ -15,13 +15,25 @@
 namespace osquery {
 namespace tables {
 
+const char* WEEKDAYS[] = {"Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"};
```

Comment:
Mind switching this to a string vector, and using something like `kTimeWeekdays`? https://github.com/facebook/osquery/blob/master/osquery/tables/system/darwin/launchd.cpp#L27

Though you might be able to grab the weekday string from C++'s `asctime`'s formats... maybe? I wouldn't search to long, a const vector is cool for now.